openapi/hubspot.crm.companies/Module.md

```diff
@@ -0,0 +1,68 @@
+## Overview
+HubSpot is a powerful easy to use Contact Mangement(CRM), email marketing, live chat, forms and analytics platform. Its 
+
+Ballerina connector for HubSpot CRM allows easy integration with HubSpot REST API via Ballerina language. 
+
+This module supports HubSpot REST API `v3` version.
+ 
+## Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+* Create a HubSpot developer account
+* Obtain tokens
+    - Use [this](https://developers.hubspot.com/docs/api/working-with-oauth4) guide to obtain the credentials which are needed to create the <CLIENT_ID> and <CLIENT_SECRET>
+
```

Comment:
The last step was missing under Prerequisites
* Configure the connector with obtained tokens